src/core/plugins/spec/actions.js

```diff
@@ -195,12 +178,7 @@ export const executeRequest = (req) => ({fn, specActions, specSelectors}) => {
   // if url is relative, parseUrl makes it absolute by inferring from `window.location`
   req.contextUrl = parseUrl(specSelectors.url()).toString()
 
-
-  if(op && op.operationId) {
-    req.operationId = op.operationId
-  } else if(op && pathName && method) {
-    req.operationId = fn.opId(op, pathName, method)
-  }
+  req.operationId = fn.opId(op, pathName, method)
```

Comment:
@ponelat we do support finding the operation by one of three operationId strategies: https://github.com/swagger-api/swagger-js/blob/f00656ac9195dc13a4df4dcfab3634f1bfa218db/src/helpers.js#L44